<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyClassClass</class>
 <widget class="QMainWindow" name="MyClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyClass</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/MyClass/background.png);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
background-repeat: no-repeat;
background-image: url(:/MyClass/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
image: url(:/MyClass/scan.png);
image-position: left;
padding: 15px;
background-color: rgb(241, 241, 241);
</string>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="whatsThis">
     <string extracomment="Close"/>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>Antivirus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
background-color: rgb(241, 241, 241);
image: url(:/MyClass/update.png);
image-position: left;
padding-left: 10px;</string>
    </property>
    <property name="text">
     <string>  Database Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>150</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
image: url(:/MyClass/card.png);
background-color: rgb(241, 241, 241);
image-position: left;
padding-left: 15px;
</string>
    </property>
    <property name="text">
     <string>Safe Money</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
image: url(:/MyClass/protection.png);
background-color: rgb(241, 241, 241);
image-position: left;
padding-left: 15px;

</string>
    </property>
    <property name="text">
     <string>Protection For 
 All Devices</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
image: url(:/MyClass/parental.png);
background-color: rgb(241, 241, 241);
image-position: left;
padding-left: 15px;

</string>
    </property>
    <property name="text">
     <string>  Parental Control</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
image: url(:/MyClass/privacy.png);
background-color: rgb(241, 241, 241);
image-position: left;
padding-left: 15px;
</string>
    </property>
    <property name="text">
     <string>   Privacy Protection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
background-color: rgb(241, 241, 241);</string>
    </property>
    <property name="text">
     <string>More Tools</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Your Protection Is Live Now</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
image: url(:/MyClass/guard.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
background-color: rgb(241, 241, 241);
</string>
    </property>
    <property name="text">
     <string>Details</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myclass.qrc"/>
 </resources>
 <connections/>
</ui>
